cmd/minishift/cmd/config/util_test.go

```diff
@@ -57,10 +57,10 @@ func TestSetInt(t *testing.T) {
 	}
 	val, ok := minikubeConfig["cpus"].(int)
 	if !ok {
-		t.Fatalf("Type is not set to int")
+		t.Fatal("Type is not set to int")
```

Comment:
So we should not use `Fatalf` because it formats the message, and we are not doing any formatting here. Is that correct?

Reply:
Correct. Usually, the 'f' versions of these type of functions do interpolation. The idea behind here is that you are using 'f', but not interpolate. You might have missed something. On the other hand, if you really just have a simple string, use the non 'f' version to make your intention clear.